pkg/plugins/policies/core/xds/endpoints.go

```diff
@@ -38,6 +38,29 @@ func GatherEndpoints(rs *xds.ResourceSet) EndpointMap {
 	return em
 }
 
+func GatherSplitEndpoints(rs *xds.ResourceSet) EndpointMap {
+	em := EndpointMap{}
+	for _, res := range rs.Resources(envoy_resource.EndpointType) {
+		if !tags.IsSplitCluster(res.Name) {
+			continue
+		}
+
+		cla := res.Resource.(*endpointv3.ClusterLoadAssignment)
+		em[cla.ClusterName] = append(em[cla.ClusterName], cla)
+	}
+	for _, res := range rs.Resources(envoy_resource.ClusterType) {
```

Comment:
This for is for clusters with endpoints that are not delivered through EDS?

Reply:
Yes